user_test.go

```diff
@@ -28,33 +28,61 @@ func TestClient_MuteUser(t *testing.T) {
 	c := initClient(t)
 	initChannel(t, c)
 
-	user := randomUser()
+	assert.GreaterOrEqualf(t, len(testUsers), 1, "not enough test users: %+v", testUsers)
```

Comment:
it won't stop the test if fails so you can still have panics so if it's a hard requirement like length check, prefer `require`.